.github/workflows/sync-higgs-shop-data-plan.yml

```diff
@@ -0,0 +1,56 @@
+name: Sync Higgs Shop Data Plan
+
+on: [ workflow_dispatch ]
+
+jobs:
+    fetch-data-plan:
+      name: Fetch Data Plan
+      uses: mParticle/mparticle-workflows/.github/workflows/data-plan-fetch.yml@stable
+      with:
+        data_plan_id: higgs_shop_basic_data_plan
+        data_plan_version: 1
+        app_relative_path: core-sdk-samples/higgs-shop-sample-app
+      secrets:
+        WORKSPACE_ID: ${{ secrets.HIGGS_SHOP_WORKSPACE_ID }}
+        CLIENT_ID: ${{ secrets.HIGGS_SHOP_CLIENT_ID }}
+        CLIENT_SECRET: ${{ secrets.HIGGS_SHOP_CLIENT_SECRET }}
+
+        
+    open-pull-request:
+      name: Open Pull Request for Data Plan
+      runs-on: ubuntu-latest
+      needs: fetch-data-plan
+      env:
+        GIT_AUTHOR_NAME: mparticle-bot
+        GIT_AUTHOR_EMAIL: developers@mparticle.com
+        GIT_COMMITTER_NAME: mparticle-bot
+        GIT_COMMITTER_EMAIL: developers@mparticle.com
+      steps:
+        - name: Checkout
+          uses: actions/checkout@v2
+          
+        - name: Download Data Plan Artifacts
+          uses: actions/download-artifact@v3
+          with:
+            name: higgs-shop-dataplan
+            path: core-sdk-samples/higgs-shop-sample-app/dataplans
+
+        - name: Create Commit if Data Plan Changed
+          uses: stefanzweifel/git-auto-commit-action@v4
+          with:
+            commit_message: Update Higgs Shop Data Plan
```

Comment:
i'd add our semantic version prefix here

Reply:
Good catch. Should it be `chore`?